src/views/PlatformOnboardingView/PlatformOnboardingView.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useContext } from 'react'
+import { Button, ButtonVariant } from '../../components/Button'
+import { Container } from '../../components/Container'
+import { OnboardingCalendar } from '../../components/OnboardingCall'
+import { PlatformOnboardingAccounts } from '../../components/PlatformOnboardingAccounts'
+import { PlatformOnboardingForm } from '../../components/PlatformOnboardingForm'
+import { PlatformOnboardingGetStarted } from '../../components/PlatformOnboardingGetStarted'
+import { PlatformOnboardingGuide } from '../../components/PlatformOnboardingGuide'
+import { PlatformOnboardingProgress } from '../../components/PlatformOnboardingProgress'
+import { PlatformOnboardingSchedule } from '../../components/PlatformOnboardingSchedule'
+import { PlatformOnboardingSummary } from '../../components/PlatformOnboardingSummary'
+import { Text } from '../../components/Typography'
+import { View } from '../../components/View'
+import { PlatformOnboardingContext } from '../../contexts/PlatformOnboardingContext'
+import { usePlatformOnboarding } from '../../hooks/usePlatformOnboarding'
+
+interface StringOverrides {
+  header?: string
+}
+
+export interface PlatformOnboardingViewProps {
+  stringOverrides?: StringOverrides
+}
+
+export const PlatformOnboardingView = (props: PlatformOnboardingViewProps) => {
+  const platformOnboarding = usePlatformOnboarding()
+  return (
+    <PlatformOnboardingContext.Provider value={platformOnboarding}>
+      <PlatformOnboarding {...props} />
+    </PlatformOnboardingContext.Provider>
+  )
+}
+
+const PlatformOnboarding = ({
+  stringOverrides,
+}: PlatformOnboardingViewProps) => {
+  const { currentStep, prevStep, nextStep } = useContext(
+    PlatformOnboardingContext,
+  )
+
+  const renderStep = () => {
+    switch (currentStep) {
+      case 0:
+        return (
+          <PlatformOnboardingGetStarted stringOverrides={stringOverrides} />
+        )
+      case 1:
+        return <PlatformOnboardingForm />
+      case 2:
+        return <PlatformOnboardingAccounts />
+      case 3:
+        return <PlatformOnboardingSchedule />
+      case 4:
+        return <PlatformOnboardingSummary />
+      default:
+        return <></>
+    }
+  }
+
+  return (
+    <View
```

Comment:
It would be good to give some `max-width` somewhere here or below to avoid this on large screens:

![image](https://github.com/user-attachments/assets/b2eab910-5517-4ac5-ad15-63a1f20e1a96)